pkg/cri/server/container_update_resources_linux_test.go

```diff
@@ -20,7 +20,7 @@ import (
 	"context"
 	"testing"
 
-	"github.com/golang/protobuf/proto"
+	"github.com/golang/protobuf/proto" // nolint:staticcheck
```

Comment:
It would be better to explain the reason in the comment. See https://golangci-lint.run/usage/false-positives/ for the format.

Reply:
It complains about github.com/golang/protobu/proto being deprecated and suggests using protobuf from google. Will add a comment. Thanks